packages/medusa/package.json

```diff
@@ -45,12 +46,14 @@
     "test:unit": "jest"
   },
   "peerDependencies": {
+    "@medusajs/types": "^0.0.1",
```

Comment:
Todo: rm, everywhere it is in peer it has to be removed from peer and only be in dev deps